specification/quantum/data-plane/Microsoft.Quantum/preview/2021-05-06-preview/quantum.json

```diff
@@ -0,0 +1,757 @@
+        }
+      }
+    },
+    "QuotaList": {
+      "description": "List of quotas.",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Quota"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link to the next page of results."
+        }
+      }
+    },
+    "Quota": {
+      "description": "Quota information.",
+      "type": "object",
+      "properties": {
+        "dimension": {
+          "type": "string",
+          "description": "The name of the dimension associated with the quota."
+        },
+        "scope": {
+          "type": "string",
+          "description": "The scope at which the quota is applied.",
+          "enum": [
+            "Workspace",
+            "Subscription"
+          ],
+          "x-ms-enum": {
+            "name": "DimensionScope",
+            "modelAsString": true
+          }
+        },
+        "providerId": {
+          "type": "string",
+          "description": "The unique identifier for the provider."
+        },
+        "utilization": {
+          "type": "number",
+          "description": "The amount of the usage that has been applied for the current period."
+        },
+        "holds": {
+          "type": "number",
+          "description": "The amount of the usage that has been reserved but not applied for the current period."
+        },
+        "limit": {
+          "type": "number",
+          "description": "The maximum amount of usage allowed for the current period."
+        },
+        "period": {
+          "type": "string",
+          "description": "The time period in which the quota's underlying meter is accumulated. Based on calendar year. 'None' is used for concurrent quotas.",
+          "enum": [
+            "None",
+            "Monthly"
+          ],
+          "x-ms-enum": {
+            "name": "MeterPeriod",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "restError": {
+      "type": "object",
+      "description": "Error information returned by the API",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorData"
+        }
+      }
+    },
+    "ErrorData": {
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "An identifier for the error. Codes are invariant and are intended to be consumed programmatically."
+        },
+        "message": {
+          "type": "string",
+          "description": "A message describing the error, intended to be suitable for displaying in a user interface."
+        }
+      },
+      "description": "An error response from Azure."
+    },
+    "PatchRequest": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/PatchDocument"
+      }
+    },
+    "PatchDocument": {
```

Comment:
Since this type represents the standard json-patch-document, can the model name more specific? e.g. "JsonPatchDocument"?

Reply:
Can change that